.github/workflows/e2e-tools.yml

```diff
@@ -62,6 +62,8 @@ jobs:
     - name: E2E Tests
       run: |
         export KUBECONFIG=${HOME}/.kube/config
+        kubectl create -f doc/examples/mcp.yaml
+        kubectl create -f doc/examples/nrop.yaml
```

Comment:
> These tests depend on running on a *un*configured cluster.

to clarify, the operator is installed but the crs aren't configured, is that right?

regarding lines 65-66 should we wait for the creation of the mcp and nrop?

Reply:
correct. We need the CRD (definition) but no CR instances.
The creation of CRDs is usually very fast so I'm cutting a corner here ( :) ) but yes, the proper way would be to wait for it.
Do we want to have this change?

Reply:
I see. Then I agree on keeping it as is now.